src/modules/Data.js

```diff
@@ -1,88 +1,97 @@
-const backend_base = process.env.NEXT_PUBLIC_BACKEND_BASE_URL
-
+const backend_base = process.env.NEXT_PUBLIC_BACKEND_BASE_URL;
 
 export async function getSpecies(speciesId, token) {
     const result = await fetch(`${backend_base}/species/${speciesId}`, {
-        'method': 'GET',
-        'headers': {
-            Authorization: `Bearer: ${token}`
-        }
-    })
-    if (!result.ok) return -1
-    return await result.json()
+        method: 'GET',
+        headers: {
+            Authorization: `Bearer: ${token}`,
+        },
+    });
+    if (!result.ok) return -1;
+    return await result.json();
+}
+
+export async function getReqSpecies(species, token) {
+    const result = await fetch(`${backend_base}/species?species=${species}`, {
+        method: 'GET',
+        headers: {
+            Authorization: `Bearer: ${token}`,
+        },
+    });
+    if (!result.ok) return result.response;
+    return await result.json();
 }
 
 export async function postSpecies(speciesJSON, token) {
     const result = await fetch(`${backend_base}/species/`, {
-        'method': 'POST',
```

Comment:
I'm confused where these changes r coming from ngl. what did u purposefully change in this doc?

Reply:
i think these changes were from Jarod's merge from last week. i just added getReqSpecies.

Reply:
oh, i see, it was the prettier formatting that auto added commas and semicolons